components/Cell.js

```diff
@@ -281,6 +332,29 @@ class Cell extends Component {
 }
 
 const styles = StyleSheet.create({
+
+
+
+  // cell左图片
```

Comment:
Could you please write your comments in english, so that everyone could understand them?